src/ext/line-numbers.lisp

```diff
@@ -15,13 +17,21 @@
 (defvar *previous-relative-line* nil)
 
 (defvar *initialized* nil)
-(defvar *line-number-format* nil)
+
+(define-editor-variable line-number-format "~6D "
+  "Set to desired format, for example, \"~2D \" for a
+two-character line-number column.")
+
+(define-editor-variable current-line-display-function
+  (lambda () (line-number-at-point (current-point)))
```

Comment:
compute-line holds the point variable.
Wouldn't it be better to take that as an argument?
It seems to me that using what is passed as an argument is a better style than thinking about what position the current-point is actually in.

```
#'line-number-at-point
```

Reply:
okay, i think i see what you mean.  the thing is i want to be able to set the value to `0` or `->`, etc.  so i'm guessing you mean make a variable like `custom-current-line-indicator` and set it to nil, and if it is defined, then `compute-line` will call that.  otherwise, show the current line number.

that does make sense.  am i understanding that correctly ?

Reply:
Okay, I understand your intention.